### ISSUE TITLE
Image Enhancement Pack revision

### ISSUE THREAD

**Issue by Badore90** [elishacloud/Silent-Hill-2-Enhancements#385]
I read you have plans for a revision of some images, so I'd like to help with some (constructive I hope) criticism. A little premise first:

Silent hill 2 is in fact a masterpiece, one of the best "interactive virtual works". And this EE is a masterpiece of a remaster, overally better than most official HD remasters.

I never saw your work as a "just for fun" mod, like i.e. some GTA SA mods that have been made by novice modders as an exercise, or just for doing funny modifications...
As the name EE suggests this is an Enhancement of one of the gratest "games" of all times, not a superficial/just for mod.

So, being this work quite serious, like an official one as intent/aim. I imagined, once the EE will be 100% complete (or at least all major updates done), one or more of the best videogame publications doing a serious, objective, professional review of it.
They will search for both pros and cons.

Let's get to the point.

The new Mary's photo is without a doubt a good work in itself. But that said, I think it isn't a objective improvement over the original comparing them on the 2 base aspects:

Technical quality:
- original photo is SH2 CG (computer graphics/FMV) quality.
- new photo is in high output resolution, but as base detail is lower than the og, more like in game cut-scene quality.

Art design:
- original Mary photo is team Silent state of the art. You can notice some kind of awareness on her eyes I think... Also she's visibly tired.
- new photo is simply different, another nature, temper, another person.

I'm not a Photoshop expert at all, so I don't know how to actually help. I see sometimes my sister drawing on PC with a graphics tablet. She use a program that allow to use transparency on images for trace them and do precise modifications or remaking, but I'm sure you're well aware of these things.

I don't know if it's possible for you to start from the original photo:

![image](https://user-images.githubusercontent.com/44087144/107993624-2a1d0d00-6fdb-11eb-9852-d5c29ae6f18f.png) 
![image](https://user-images.githubusercontent.com/44087144/107996392-93534f00-6fe0-11eb-908b-36b96a73bd25.png)

retouching it to just adapt the detail for a high resolution output, making sure to keep the original art and technical quality.

Another example is the old man coin.
That's technically good I think, but different art design.

I didn't reviewed all the images. Many I've seen seems very good to me, but I'd need to do a direct comparison for a more precise evaluation.

Imho, Mary's photo is the most important. I think that adjusting some of those images, a eventual serious, honest, reviewer from a official newspaper won't find anything to criticize on this EE work, but only "Enhancements".

**Comment by Polymega** [comment]
Yes, the next project update will have some revisions to the high res full screen images. There are just a couple that are majorly changed, the rest are minor adjustments/tweaks. More information will be shared on those major adjustments closer to or on the day of our next update release.

As for Mary's photo: The upscaler I used for this part of the project did not upscale her photo to what I consider a satisfactory result. The upscaler did well for _most_ of the grungy/environmental images, but not organic images (such as people). Its results were too much of a mess for me to work with. 

![image](https://user-images.githubusercontent.com/27395260/108008084-7feab880-6fcd-11eb-9275-47078279037f.png)

However, a few weeks ago I had success using Remini on an image of Maria, so I can certainly entertain the idea of putting Mary's photo through Remini as well and see what comes of it. If the base results impress me as much as they did for Maria's image, I can use the base results and further refine the image to make a revised version of her photo.

For images such as the coin, the original image is already very rough in quality, so upscaling doesn't really do anything for it, And without the source image, a reference image needs to be used in its place for a high res version. There's no other way around that.

![image](https://user-images.githubusercontent.com/27395260/108008587-b1b04f00-6fce-11eb-8069-eaa2d1cdc513.png)

**Comment by Badore90** [comment]
> Yes, the next project update will have some revisions to the high res full screen images. There are just a couple that are majorly changed, the rest are minor adjustments/tweaks. More information will be shared on those major adjustments closer to or on the day of our next update release.
> 
> As for Mary's photo: The upscaler I used for this part of the project did not upscale her photo to what I consider a satisfactory result. The upscaler did well for _most_ of the grungy/environmental images, but not organic images (such as people). Its results were too much of a mess for me to work with.
> 
> ![image](https://user-images.githubusercontent.com/27395260/108008084-7feab880-6fcd-11eb-9275-47078279037f.png)
> 
> However, a few weeks ago I had success using Remini on an image of Maria, so I can certainly entertain the idea of putting Mary's photo through Remini as well and see what comes of it. If the base results impress me as much as they did for Maria's image, I can use the base results and further refine the image to make a revised version of her photo.
> 
> For images such as the coin, the original image is already very rough in quality, so upscaling doesn't really do anything for it.
> 
> ![image](https://user-images.githubusercontent.com/27395260/108008587-b1b04f00-6fce-11eb-8069-eaa2d1cdc513.png)
> 
> And without a high quality source image of the old man, a reference image (a different old man) needs to be used in its place for a high res version. There's no other way around that. The light source hitting the coin is different in the high res version, but that's mostly due to working around how the light was hitting the old man in the new reference image I used.
> 
> But between something like a coin and Mary's photo, I will agree that Mary's photo is notably more important to upscale well.

Thank you for the answer

I think this Mary's photo you posted is actually good (considering the fact that it is only visible from menu screen and it's little, plus it's a quite old photo that James carries with him, etc... So you won't notice those little imperfections, it is much better than the original without changing it). 
Some adjustments can be made but, really, I'm 99% sure no one would have complained about this photo.

I see the old man coin is a difficult one... He has a goatee. It also seems he have something on his head watching oh the upper back. 

I could show it to my sister to see what she think (she's very good at drawing).

**Comment by Badore90** [comment]
My sister retouched a little the photo you posted (I said to her to modify the less possible), she re-draw the hair ('cause they were  "blocky" pixelated, the mouth and corrected other details). It's just a first example, here:

<img width="450" alt="Mary's photo PROVA4" src="https://user-images.githubusercontent.com/44087144/108355307-f1ec1900-71ea-11eb-8963-c23879531bd6.png">

What do you think?

**Comment by Badore90** [comment]
Sorry I uploaded while she was still adjusting it. They finished the shadows to be more similar to the original and some other minor adjustments:

<img width="450" alt="Mary's photo PROVA6" src="https://user-images.githubusercontent.com/44087144/108362627-4778f380-71f4-11eb-9fab-a200689ebf0c.png">

**Comment by Badore90** [comment]
Just for information: My sister is very good in real/handmade designs (landscapes, animals, but especially portraits, sometimes they're indistinguishable from the real counterpart), she colors either with tempera colors, watercolors, oil paints. 
They gave her a graphic tablet some months ago, so she done some practice with it also.

She's also passionate about photography (she has a little collection af analog cameras, some very old but good, she don't like much digital cameras).

I see this is just a cutout not the actual image (the white on the picture frame isn't as white as it should, it seems a little desaturated), the work should be made on the original file.

We both have the same feeling about the photo being even too sharp. 
Because it's clear they intended the photo to look like it's been made with a low quality polaroid. Regardless of the resolutions, the original photo is overexposed, almost burnt in whites (this was even more accentuated on CRTs due to the white bloom), also according with the story's meaning (Mary is a so bright and thus simultaneously clouded memory...). 

So she thought to apply a filter for the photo to be more authentic (just an example):

![Mary's photo PROVA6-01](https://user-images.githubusercontent.com/44087144/108413018-c8eb7880-722a-11eb-8a93-4dcf0786d483.jpeg)

Also, pratically every image on the game, more or less, have similar opacity filter. Since outside there's the fog and within buildings there's the dust. Take Eddie's FMVs as an example. The filter really add realistic three-dimensional volume and in fact more realistic/volumetric illumination.

It's difficult to notice in 2d images due to the low res. but watching carefully I think is present in most (if not all) of them, that layer of opacity some people are complaining the absence in some new images.
Also on some original images there's applied some depth of field filter (like in the signboard one) that also add to the three-dimensionality.

Coming back to Mary's photo. The last one with the heavy filter I think is too much different from the original, more real in some ways but it's changed and it won't please fans, it was only an example.

We eventually made a comparison with the original (NTSC-U version, playing from the Ps2 connected to a CRT Tv with scart-RGB cable).
On Ps2-CRT the photo was a little mpre overexposed and Mary's saturation stands out more (the background is pratically black & white), like this (she adjusted this photo from the smartphone, don't judge the little details):

![Mary's photo PROVA6-02](https://user-images.githubusercontent.com/44087144/108415179-7c556c80-722d-11eb-88cd-c81eb7ba42dd.jpeg)

But as i was saying it can be caused by the photo not being the original file, so probably colors are desaturated. The original file should be as the ps2 one.

Sorry for the lenght '^^

**Comment by Polymega** [comment]
Hi Badore,

Please thank your sister for taking the time to provide her thoughts and work/examples on the photo. She's very talented!

> that layer of opacity some people are complaining the absence in some new images.

The original images also used TV levels (16-235) and were changed to full levels (0-255), so that might have something to do with it. This was intentional, to give full blacks to images that use blacks. TV levels were used back in the day because, well, the images were shown on TVs and old TVs were incapable of displaying a full level range.

> Also on some original images there's applied some depth of field filter (like in the signboard one) that also add to the three-dimensionality.

Yes, I've done the same. Here is the photo with the foreground layers disabled. 

![p_hanging_2](https://user-images.githubusercontent.com/27395260/108417567-21585100-71fe-11eb-93f2-8dc8664ae639.jpg)

It's a shame this apparently wasn't noticed. I think it might go back to upscaling the images will never please everyone. And I have to be okay with that. I've further increased the three-dimensionality to many of the images, adding additional highlights and shadows to further accentuate their depths. 

![before](https://user-images.githubusercontent.com/27395260/108418356-2073ef00-71ff-11eb-911b-8de8f85a830b.gif)

That's just one example. Again, I'm sad this apparently hasn't been noticed and that people might think I might have done the opposite instead. 

I will look into Mary's photo some more though. Thank you for your thoughts, and please thank your sister again.

**Comment by Badore90** [comment]
> Hi Badore,
> 
> Please thank your sister for taking the time to provide her thoughts and work/examples on the photo. She's very talented!

Thank you. Also she said the same about you (I showed her the #6 video about the remade images). As I said she always draw real painting, she's not much experience with PC programs and wouldn't be able to do such 3d images.

She played Silent hill 2 only once, on Xbox, years ago. She mostly forgot the story but remember the game's atmosphere and spirit.

> > that layer of opacity some people are complaining the absence in some new images.
> 
> The original images also used TV levels (16-235) and were changed to full levels (0-255), so that might have something to do with it. This was intentional, to give full blacks to images that use blacks. TV levels were used back in the day because, well, the images were shown on TVs and old TVs were incapable of displaying a full level range.

I don't think so, original images seem well scaled to 0-255. 

I was referring to some filter that simulate volumetric light on 2d images. Without it images would be sharper but unrealistic. 
But I'm mostly referring about comments I read on YT for this matter (sorry I haven't had much time to play through the game). So it's just an advice in case you didn't thought about it.

Allow me a side note:
Because I know it can happen that a sharp image with razor detail seems good and applying a hazy filter seems to decrease the detail. But this isn't true, as you can see from the example on the filtered mary's photo (you can still see the detail on it, the filter just add more realism). 
Detail and sharpness are 2 completely different things. 

Sharpness is the evil! It works increasing the micro-contrast by adding white contours around the original lines making the  detail to pop-out.

Firtly this isn't realistic, since in reality the detail doesn't pop-out that way, you're falsing the original picture.
Secondly you're loosing actual detail, since when you're displaying a native res. material (i.e. a 1920x1080 BD movie on a 1920x1080 TV) the pixels used for making the white contourns can not be used for showing the actual color the pixel should have (since it's white), so you're making some detail to pop-out, loosing the remaining part.

It's like with saturation, increasing it seems to add vividness to the colors at a first glance, but you're actually just loosing color shades and realism.

> > Also on some original images there's applied some depth of field filter (like in the signboard one) that also add to the three-dimensionality.
> 
> Yes, I've done the same. Here is the photo with the foreground layers disabled.
> 
> ![p_hanging_2](https://user-images.githubusercontent.com/27395260/108417567-21585100-71fe-11eb-93f2-8dc8664ae639.jpg)

Sorry, I didn't re-checked the picture. I wasn't sure there was the depth of field because the picture background is very dark (is much easier to norice the effect on the og image since is much less dark).
 
> It's a shame this apparently wasn't noticed. I think it might go back to upscaling the images will never please everyone. And I have to be okay with that. I've further increased the three-dimensionality to many of the images, adding additional highlights and shadows to further accentuate their depths.
> 
> ![before](https://user-images.githubusercontent.com/27395260/108418356-2073ef00-71ff-11eb-911b-8de8f85a830b.gif)
> 
> That's just one example. Again, I'm sad this apparently hasn't been noticed and that people may think I might have done the opposite instead.

Wait wait, don't misunderstand me. Those details are noticeable. You made a great work this is out of discussion.

I'm simply looking for eventual oversights (that can always happen) and/or ways to further improve some of the images, that's all. Most of what you done is just perfect, but what's good is good, we have to talk about the less goo/improvable things ;)

About the fact to cannot please anyone, Imho that's true when doing a new game:
You have an idea, you accomplish it/publish the game, most probably it won't like to everyone.

But here we are enhancing an already existing game that people love. Critics are coming from fans of the game.
I think the difficult (very difficult) thing is to distinguish objective improvements from personal taste changes.

Doing the first should please anyone (there could be some misunderstandings at most, like the recent 4:3 FMV discussion, that can easilly be cleared). 
Basically you already done it since the start. Indeed anyone able to use his brain couldn't complain too much about EE changes, since original game's data is untouched and you can select what to enable/disable from the ini and/or renaming some folders (this is really a brilliant idea you don't fins in all mods).

But I think complaints are good as input for improving the mod, anyway. And the best thing would be having anyone pleased enabling all the EE features.

Indeed, since it is the first time it have been made a quite huge modification, maybe it'd be good to have a option in the ini for it for who don't like much the first hd image release and/or play the game in low res. (because renaming the folder can be a little more complicated for some people).

That said, judging by the ctriticism on YT, with some changes (adding dirt, maybe some filters also) on certain images, the hd images will just please everyone.

> I will look into Mary's photo some more though. Thank you for your thoughts, and please thank your sister again.

Thank you, she was pleased to help. She said to just ask if her help is needed.

**Comment by Badore90** [comment]
For example take this image:

![image](https://user-images.githubusercontent.com/44087144/108547722-2941ef00-72eb-11eb-9c2d-c0e664d1629f.png)

![image](https://user-images.githubusercontent.com/44087144/108547792-3ced5580-72eb-11eb-9362-22c3d7260fa1.png)

Maybe some filter can help (togheter with some more dirt and a little more ruined sheets) making the image more authentic, less cartoony, more silent hill style as the original indeed (little details make the difference). 

You could test some filters to see how it looks. Take my suggestion as what it is, just a testing idea, it is possible the results won't be good. Just don't rule out the possibility of adding more filters, for making the images as hazy and dilapidated as possible, otherwise there's a little too clear separation between the re-made images and the up-scaled ones.

**Comment by Polymega** [comment]
Just like Maria's save background image, I have always planned at some point to revisit Mary's photo as I was never truly satisfied with it (the in-game model was moreso used as a placeholder). But until that time came, I had to wait until I found new methods to accomplish what I was after. 

As earlier mentioned, Remini produced desirable results for Maria's face--good enough for me to properly revisit the image. With this new software at my disposable, I also ran Mary's photo through Remini and received good results; good enough for me to work from. 

![temp1](https://user-images.githubusercontent.com/27395260/108617871-891dc000-73e7-11eb-9a2e-031dcd20b64c.png)

This shows Remini's results, then my adjustments after:

![compare4](https://user-images.githubusercontent.com/27395260/108617995-722b9d80-73e8-11eb-8daa-f5c894850be7.gif)

At this point, I'm satisfied with the image.

![comapre3](https://user-images.githubusercontent.com/27395260/108617479-b9b02a80-73e4-11eb-87a1-cc651eceb820.gif)

**Comment by Badore90** [comment]
> Just like Maria's save background image, I have always planned at some point to revisit Mary's photo as I was never truly satisfied with it (the in-game model was moreso used as a placeholder). But until that time came, I had to wait until I found new methods to accomplish what I was after.
> 
> As earlier mentioned, Remini produced desirable results for Maria's face--good enough for me to properly revisit the image. With this new software at my disposal, I also ran Mary's photo through Remini and received good results; good enough for me to work from.
> 
> ![temp1](https://user-images.githubusercontent.com/27395260/108617871-891dc000-73e7-11eb-9a2e-031dcd20b64c.png)
> 
> This shows Remini's results, then my adjustments after:
> 
> ![compare4](https://user-images.githubusercontent.com/27395260/108617995-722b9d80-73e8-11eb-8daa-f5c894850be7.gif)
> 
> At this point, I'm satisfied with the image.
> 
> ![comapre3](https://user-images.githubusercontent.com/27395260/108617479-b9b02a80-73e4-11eb-87a1-cc651eceb820.gif)

I've something to talk about in private, if for you is not a problem.

**Comment by Polymega** [comment]
Uh, sure. You can send me a message through here: http://ratio.townofsilenthill.com/about/#contactme

**Comment by Badore90** [comment]
There was something I thought to talk about but I changed my mind. Thank you for the link anyway.

**Comment by Polymega** [comment]
I think most points have been discussed and/or addressed. I'll close this ticket out. Thanks for your notes!
